packages/cmf/src/cmfConnect.js

```diff
@@ -290,40 +285,27 @@ export default function cmfConnect({
 
 			onEventDispatch(props, key) {
 				if (CONSTANT.IS_HANDLER_DISPATCH_REGEX.test(key)) {
-					const handlerKey = key.replace(CONSTANT.IS_HANDLER_DISPATCH, '');
 					props.toOmit.push(key);
```

Comment:
those 4 lines looks very similar on the 3 cases, a little function used like setHandler(props, expressionSuffix, getHandler) can shorten the code a little bit